src/Units/ArmyWrappers/HydraliskWrapper.cpp

```diff
@@ -3,5 +3,14 @@
 void HydraliskWrapper::onFrame() {}
 
 void HydraliskWrapper::displayInfo() {
+    if (this->unit->getLastCommand().getTarget()) {
+        BWAPI::Broodwar->drawLineMap(this->unit->getPosition(), this->unit->getLastCommand().getTarget()->getPosition(), BWAPI::Colors::White);
+    }
+    else if (this->unit->getLastCommand().getTargetPosition().isValid()) {
+        BWAPI::Broodwar->drawLineMap(this->unit->getPosition(), this->unit->getLastCommand().getTargetPosition(), BWAPI::Colors::White);
+    }
+    else if (this->unit->getLastCommand().getTargetTilePosition().isValid()) {
+        BWAPI::Broodwar->drawLineMap(this->unit->getPosition(), BWAPI::Position(this->unit->getLastCommand().getTargetTilePosition()), BWAPI::Colors::White);
+    }
 	BWAPI::Broodwar->drawTextMap(this->unit->getPosition(), "HydraliskWrapper");
```

Comment:
all of these wrapper class name outputs can be removed now, they were just placeholders until better information could be output, and were validation that the proper wrapper class was being used